hawc/apps/study/templates/study/study_detail.html

```diff
@@ -56,18 +56,25 @@ <h2 class="d-inline-block">{{object}}</h2>
 
   <div id="study_details"></div>
 
-  {% if obj_perms.edit %}
+  {% if obj_perms.edit and internal_communication %}
```

Comment:
once a communication is added, Internal Communications is always listed on the detail page--even if it is updated to say nothing. Not really an issue but it could be changed by adding `and internal_communication != "<p><br></p>"` to the if statement above, and for the corresponding assessment_detail template if statement